library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -231,6 +231,16 @@ public void onMapClick(LatLng latLng) {
     polyline.removeFromGoogleMap();
   }
 
+  @Override
+  public void startTrackingUserLocation() {
```

Comment:
We already have a `setMyLocationEnabled` method in the `NativeGoogleMapFragment`. We should add support for that method in AirMapView and the WebViewMapFragment (using your logic, just renaming the method calls)